### ISSUE TITLE
Error in lsp-emacs upon formatting any Elixir file: (wrong-type-argument integerp 2.0)

### ISSUE THREAD

**Issue by x-ji** [elixir-lsp/elixir-ls#255]
I suspect this could have something to do with this change: https://github.com/elixir-lsp/elixir-ls/commit/6fb7164f35855013d2b2b76dc8bbedb087184df2

### Environment

* Elixir & Erlang versions (elixir --version):
```
Erlang/OTP 22 [erts-10.6.2] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [hipe]

Elixir 1.10.2 (compiled with Erlang/OTP 22)
```
* Operating system: MacOS 10.15.4
* Editor or IDE name (e.g. Emacs/VSCode): Emacs
* LSP Client name: `lsp-mode` (Emacs)

### Troubleshooting

- [ x ] Restart your editor (which will restart ElixirLS) sometimes fixes issues
- [ x ] Stop your editor, remove the entire `.elixir_ls` directory, then restart your editor
  * NOTE: This will cause you to have to re-run the dialyzer build for your project

### Logs

1.  If using a client other than VS Code, please try VSCode's "ElixirLS: Elixir support and debugger" extension. Does it reproduce your failure?
  No
2.  Create a new Mix project with `mix new empty`, then open that project with VS Code and open an Elixir file. Is your issue reproducible on the empty project? If not, please publish a repo on Github that does reproduce it.
  Yes. Just try to run `lsp-format-buffer` on any Elixir file.
3. In Emacs Messages buffer:

```
Debugger entered--Lisp error: (wrong-type-argument integerp 2.0)
  lsp--line-character-to-point(15 2.0)
  lsp--position-to-point(#<hash-table equal 2/65 0x61d75ca9>)
  lsp--range-to-region(#<hash-table equal 2/65 0x434e25b1>)
  lsp--apply-text-edit-replace-buffer-contents(#<hash-table equal 2/65 0x434e40b1>)
  #f(compiled-function (edit) #<bytecode 0x434e46d9>)(#<hash-table equal 2/65 0x434e40b1>)
  mapc(#f(compiled-function (edit) #<bytecode 0x434e46d9>) (#<hash-table equal 2/65 0x434e40b1>))
  lsp--apply-text-edits((#<hash-table equal 2/65 0x434e40b1>))
  lsp-format-buffer()
  funcall-interactively(lsp-format-buffer)
  call-interactively(lsp-format-buffer record nil)
  command-execute(lsp-format-buffer record)
  counsel-M-x-action("lsp-format-buffer")
  ivy-call()
  ivy-read("M-x " ("toggle-debug-on-error" "lsp-workspace-folders-remove" "lsp-workspace-restart" "describe-key" "describe-variable" "describe-function" "reindent-then-newline-and-indent" "electric-indent-mode" "smie-config-show-indent" "newline" "newline-and-indent" "describe-mode" "electric-indent-local-mode" "make-directory" "treemacs-mode" "recover-this-file" "s" "cd" "5x5" "arp" "dbx" "dig" "erc" "ert" "eww" "ftp" "gdb" "irc" "jdb" "lsp" "man" "treemacs-follow-mode" "treemacs-quit" "treemacs-refresh" "jb" "mpc" "orc" "orn" "oro" "orp" "oru" "pdb" "pwd" "rsh" "sdb" "wos" "xdb" "calc" "deer" "diff" ...) :predicate #f(compiled-function (x) #<bytecode 0x6124236d>) :require-match t :history counsel-M-x-history :action counsel-M-x-action :keymap (keymap (67108908 . counsel--info-lookup-symbol) (67108910 . counsel-find-symbol)) :initial-input nil :caller counsel-M-x)
  counsel-M-x()
  funcall-interactively(counsel-M-x)
  call-interactively(counsel-M-x nil nil)
  command-execute(counsel-M-x)

```

**Comment by x-ji** [comment]
I think it's the use of `/` instead of `div` on line 84. I'll submit a fix.

**Comment by x-ji** [comment]
Submitted a fix at https://github.com/elixir-lsp/elixir-ls/pull/250

**Comment by x-ji** [comment]
There's already another PR that fixes this at #250

**Comment by axelson** [comment]
Closing as a duplicate of #249, thanks for the report!
